README.md

```diff
@@ -302,3 +319,7 @@ pipx is maintained by a team of volunteers (in alphabetical order)
 Issues and Pull Requests are definitely welcome! Check out [Contributing](https://pypa.github.io/pipx/contributing/) to get started.
 Everyone who interacts with the pipx project via codebase, issue tracker, chat rooms, or otherwise is expected to follow
 the [PSF Code of Conduct](https://github.com/pypa/.github/blob/main/CODE_OF_CONDUCT.md).
+
```

Comment:
Empty code block?

Reply:
Good catch, not sure how that got in there.